latex/slides/07_arrays.tex

```diff
@@ -227,20 +227,21 @@ \subsection{}
 \begin{frame}[fragile]{Remember, Remember}
 	Old and busted:
 	\begin{itemize}
-		\item The size of variables may differ on different architectures
-		\item The size of a char is \textbf{always} 1 byte
+		\item Number of bytes per type may differ on different architectures
+		\item A char is 1 byte long (but can be longer in some cases)
 	\end{itemize}
 	New hotness:
 	\begin{itemize}
-		\item You can get the size of a variable with the operator \textit{sizeof}
-		\item You can also get the size of a type with sizeof
+		\item You can get the \textit{size} of a variable or type with the operator \textit{sizeof}
+		\item The \textit{size} is measured in chars
```

Comment:
I'd rather say `the size is measured in bytes`. Also, `sizeof` is an operator for variables. For types, I think it's a macro, but I'm not 100% sure about that.